ixmp/core.py

```diff
@@ -554,16 +554,21 @@ def set_list(self):
         return to_pylist(self._jobj.getSetList())
 
     def init_set(self, name, idx_sets=None, idx_names=None):
-        """initialize a new set in the scenario
+        """Initialize a new set in the scenario.
 
         Parameters
         ----------
         name : string
-            name of the item
+            Name of the item
         idx_sets : list of strings
             index set list
```

Comment:
capitalize others?

Reply:
Yes, there are many more methods to go through, yet. Still WIP.